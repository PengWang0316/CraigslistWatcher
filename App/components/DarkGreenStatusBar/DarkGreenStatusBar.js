import React from 'react';
import { StatusBar } from 'react-native';

import Colors from '../../Colors';

const DarkGreenStatusBar = () => (<StatusBar
  barStyle="light-content"
  backgroundColor={Colors.primaryDark}
/>);
export default DarkGreenStatusBar;
