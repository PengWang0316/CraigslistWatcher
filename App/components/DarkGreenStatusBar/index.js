import DarkGreenStatusBar from './DarkGreenStatusBar';

export default DarkGreenStatusBar;
