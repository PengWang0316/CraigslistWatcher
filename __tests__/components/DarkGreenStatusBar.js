import React from 'react';
import renderer from 'react-test-renderer';

import DarkGreenStatusBar from '../../App/components/DarkGreenStatusBar';

jest.mock('react-native', () => ({ StatusBar: 'StatusBar' }));

describe('DarkGreenStatusBar Test', () => {
  test('snapshot', () => expect(renderer.create(<DarkGreenStatusBar />).toJSON()).toMatchSnapshot());
});
