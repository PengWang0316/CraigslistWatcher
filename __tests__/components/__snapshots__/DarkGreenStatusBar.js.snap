// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DarkGreenStatusBar Test snapshot 1`] = `
<StatusBar
  backgroundColor="#24424e"
  barStyle="light-content"
/>
`;
